plugins/orchestrator/src/utils/formState.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  JsonArray,
+  JsonObject,
+  JsonPrimitive,
+  JsonValue,
+} from '@backstage/types';
+
+export function isJsonPrimitive(value: JsonValue): value is JsonPrimitive {
+  return (
+    typeof value === 'number' ||
+    typeof value === 'string' ||
+    typeof value === 'boolean' ||
+    value === null
+  );
+}
+
+export function isJsonObject(value: JsonValue): value is JsonObject {
+  return typeof value === 'object' && value !== null && !Array.isArray(value);
+}
+
+export function isJsonArray(value: JsonValue): value is JsonArray {
```

Comment:
you can remove this

Reply:
This fits better inside this module: `src/utils/TypeGuards.ts`